### ISSUE TITLE
button press detection hit&miss

### ISSUE THREAD

**Issue by eidermar** [matthutchinson/lifx_dash#6]
hi matt,

sorry to bother you again. for a few days I'm having problems with the button press detection. It sometimes works but most times doesn't. when inspecting packages with wireshark they seem to arrive properly as expected. what I discovered though is that sometimes the button sends 2 dhcp packages and sometimes only 1. it seems to me that detection only works when 2 dhcp packages are sent. is that possible? is there a way around this?

thanks again for your help!

**Comment by matthutchinson** [comment]
Yes! I've noticed this too.  I need to rework the gem to take account of this somehow.. I haven't had time to look into it yet.  If you get to it before me, feel free to issue a pull-request 😄

**Comment by eidermar** [comment]
As much as I would love to help, I'm a power user at best. I wouldn't even know where to start.
